.github/workflows/pr_lint_and_test.yml

```diff
@@ -56,3 +56,27 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           lcov-file: ./coverage/lcov.info
+      - name: Metrics package size report
+        uses: flochaz/pkg-size-action@v1.2.4
```

Comment:
Why did you have to fork it?

Reply:
because it does not support mono repo . I am preparing a pr to propose my changes to the original implementation. but in the mean time I didn't want to be blocked